### PR TITLE
Change checkstyle configuration

### PR DESCRIPTION
**Pull request by egedebruin**
I have changed some checkstyle configurations which should be better now. Please check if you agree with these configurations.

### CODE REVIEW

**Comment by lisv** [comment]
I like this. If everyone agrees we should start solving all the warnings.

File: src/main/java/camera/CameraChest.java [hunk 1]
```diff
@@ -17,10 +17,12 @@
  * Class for describing a chest found in a camerastream/video/image.
  */
 public class CameraChest extends CameraObject {
-    private static final Scalar BOXCOLOUR_LOWER = new Scalar(18,100,100);
-    private static final Scalar BOXCOLOUR_UPPER = new Scalar(35,255,255);
+    private static final Scalar CHESTCOLOUR_LOWER = new Scalar(18, 100, 100);
+    private static final Scalar CHESTCOLOUR_UPPER = new Scalar(35, 255, 255);
+    private static final Scalar CHESTBOXCOLOUR = new Scalar(255, 0, 255);
```
Comment (robinjo78) [inline src/main/java/camera/CameraChest.java]: Shouldn't this be chestcolour in order to be consistent?

Comment (Jasperkroes) [reply src/main/java/camera/CameraChest.java]: Did checkstyle really give a warning/error about these names?

Comment (Jasperkroes) [reply src/main/java/camera/CameraChest.java]: no, because it is not the same thing

**Pull request merged**